.github/workflows/jetbrains-updates-template.yml

```diff
@@ -59,12 +59,24 @@ jobs:
             This PR updates the ${{ inputs.productName }} IDE image to the latest release version.
 
             ## How to test
-            Go to the preferences and select ${{ inputs.productName }} as desktop IDE. Start a workspace and verify that the new IDE version works.
+            1. Open the preview environment generated for this branch
+            2. Choose the stable version of ${{ inputs.productName }} as your preferred editor
+            3. Start a workspace (for IntelliJ use this repository: `https://github.com/gitpod-io/spring-petclinic`)
```

Comment:
something like this `${{ inputs.exampleRepo }}` for now, further see https://github.com/gitpod-io/gitpod/pull/10003#discussion_r872367882

Reply:
@mustard-mh my understanding is that we don't have example repos for each IDE yet 🤔 Do we?

Reply:
intellij one got some special configurations in `.gitpod.yml` of course😄